cachito/workers/pkg_managers/general_js.py

```diff
@@ -92,13 +115,24 @@ def download_dependencies(request_id, deps, proxy_repo_url, skip_deps=None):
 
         log.info("Processing %d npm dependencies to stage in Nexus", len(deps))
         downloaded_deps = set()
+
+        # Get Cachito supported prefixes. Include the part only before ":" in order to
+        # match conveniently with the dependency version
+        known_git_prefixes = {i.rsplit(":", 1)[0] for i in get_git_prefixes()[0]}
+        other_prefixes = {i.rsplit(":", 1)[0] for i in get_git_prefixes()[1]}.union(
+            i.rsplit(":", 1)[0] for i in get_http_prefixes()
+        )
+
         # This must be done in batches to prevent Nexus from erroring with "Header is too large"
         deps_batches = []
         counter = 0
         batch_size = get_worker_config().cachito_js_download_batch_size
         for dep in deps:
+            # Initialize external dependency version
```

Comment:
This comment doesn't add much value since it's clear this is happening in the following line. If you'd like to keep the comment, you can explain why the variable is used.
```suggestion
```

Reply:
No, I was skeptical as well and felt it was unnecessary. Removing.